ui-modules/utils/quick-launch/quick-launch.js

```diff
@@ -47,11 +47,21 @@ export function quickLaunchDirective() {
     function controller($scope, $http, $location, brSnackbar) {
         $scope.deploying = false;
         $scope.model = {
-            newConfigFormOpen: false
+            newConfigFormOpen: false,
+            
+            // should never be null, so the placeholder in UI for model.name will never be used;
+            // hence autofocus is disabled
```

Comment:
The name field might be filled up, but the input itself is not disabled, and there are other fields in the form so `autofocus` should be set on one of these.

Reply:
there's a good chance that the location and the config don't need any change so in this instance autofocussing on anything could be misleading.  arguably we should check if anything is required and make the first required thing autofocus but not sure that's worth the work, and if nothing is required it's unclear to we select "deploy" (even though user might want to change non-required things) or something non-required (which would be confusing).  better IMO to have the tab set just before the first field so it's easy to select using mouse or keyboard - but let's chat.

Reply:
Fair enough, I buy that.